src/components/popover/widgets/capture-keystroke/capture-keystroke-button.component.html

```diff
@@ -1,6 +1,7 @@
 <button type="button" class="btn btn-sm btn--capture-keystroke"
             [ngClass]="{'btn-default': !record, 'btn-info': record}"
-            (click)="start()"
+            (keydown.enter)="onButtonEnterDown($event)"
```

Comment:
There are more than improvements ones are needed others don't. If you could suggest a better name for the pr than it is welcome. 
My original issue why I've started the modification of these things is because the saveScanCode is called on both keydown and up and for me it is not clear why should be called in both places. With the escape it caused me to set to escape at down and clear it on up. If you would like then maybe I could reproduce it again.